From c23ddaa64cb0bc56441d0567f6ee8f7253ababf4 Mon Sep 17 00:00:00 2001
From: Jose Lamego <jose.a.lamego@intel.com>
Date: Wed, 5 Sep 2018 16:32:41 -0500
Subject: [PATCH kubernetes] kubelet.service: add bootstrap-kubeconfig flag

The bootstrap-kubeconfig is required to enable boostrapping when
starting the kubelet.

Fixes: clearlinux/distribution#177

Signed-off-by: Jose Lamego <jose.a.lamego@intel.com>
---
 kubelet.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kubelet.service b/kubelet.service
index 28f81ef..4dd7bf1 100644
--- a/kubelet.service
+++ b/kubelet.service
@@ -3,7 +3,7 @@ Description=kubelet: The Kubernetes Node Agent
 Documentation=http://kubernetes.io/docs/
 
 [Service]
-Environment="KUBELET_KUBECONFIG_ARGS=--kubeconfig=/etc/kubernetes/kubelet.conf"
+Environment="KUBELET_KUBECONFIG_ARGS= --bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --kubeconfig=/etc/kubernetes/kubelet.conf"
 Environment="KUBELET_SYSTEM_PODS_ARGS=--pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true"
 Environment="KUBELET_NETWORK_ARGS=--network-plugin=cni --cni-conf-dir=/etc/cni/net.d --cni-bin-dir=/usr/libexec/cni"
 Environment="KUBELET_DNS_ARGS=--cluster-dns=10.96.0.10 --cluster-domain=cluster.local"
-- 
2.18.0

